### PR TITLE
James

### PR DESCRIPTION
**Pull request by zlimez**
- AboutUs update
- UG Quickstart with Ui mockup
- DG User cases

### CODE REVIEW

**Review by seelengxd — CHANGES_REQUESTED** [review]
1. For your image + PPP, the name should be **your Github username** in lower case.
1. follow commit subject conventions pls

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 
-ArtBuddy (AB) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
```
Comment (chloeelim) [inline docs/UserGuide.md:6]: Hmmm, wonder if this was an accidental mistake? Probably should use the ArtBuddy description instead?

File: docs/AboutUs.md [hunk 2]
```diff
@@ -48,3 +48,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 * Responsibilities: Logic + Testing
+
+### James Chiu
+
+<img src="images/jameschiu.jpeg" alt="jameschiu" width="200px">
```
Comment (chloeelim) [inline docs/AboutUs.md:54]: Minor note I think you need to rename your picture so it's `jameschiu.png` even though it isn't a png.

Screenshot 2022-09-28 at 10 45 14 AM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -14,27 +14,27 @@ ArtBuddy (AB) is a **desktop app for managing contacts, optimized for use via a
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `artbuddy.jar` from [here](https://github.com/AY2223S1-CS2103T-W11-3/tp/releases).
+2. Download the latest `artbuddy.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```
Comment (chloeelim) [inline docs/UserGuide.md:17]: This here too, let's maybe edit it back to our TP.

**Review by chloeelim — CHANGES_REQUESTED** [review]
Hi @zlimez, I noticed some minor stuff you might probably need to change again (sorry for the many changes requested so far):
1. You probably need to rename your profile picture so it is `zlimez.png` (use your github username and rename to end with `.png`) even though its a `jpeg` so the scripts can detect that you've uploaded your picture.
2. I think you accidentally override some changes @ngxingyu made when he updated the user guide to ArtBuddy, could you change these back?

Other than that, LGTM, thanks!

File: docs/UserGuide.md (hunk 3 above)
Comment (zlimez) [reply docs/UserGuide.md:17]: Thx will take not i will go read the instructions again before making any request so sorry

**Pull request closed without merging**